packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -94,6 +94,13 @@ export async function load_node({
 
 				opts.headers = new Headers(opts.headers);
 
+				// merge headers from request
+				for (const [key, value] of event.request.headers.entries()) {
```

Comment:
The `.entries()` is unnecessary, `headers` is already iterable

```suggestion
				for (const [key, value] of event.request.headers) {
```